apps/src/blocklyAddons/cdoTrashcan.js

```diff
@@ -0,0 +1,103 @@
+import GoogleBlockly from 'blockly/core';
+
+export default class CdoTrashcan extends GoogleBlockly.Trashcan {
+  /** Use our trash png and add ⃠ for undeletable blocks
```

Comment:
nit: This character just shows as a box?

Reply:
whoops, it shows up in my editor, but changed to "circle with line through it" (side note: I really wish that symbol had a better name)

Reply:
Some potential names at https://en.wikipedia.org/wiki/No_symbol :)